examples/structured_data/wide_deep_cross_networks.py

```diff
@@ -274,25 +273,25 @@ def encode_inputs(inputs, use_embedding=False):
             # Create a lookup to convert string values to an integer indices.
             # Since we are not using a mask token nor expecting any out of vocabulary
             # (oov) token, we set mask_token to None and  num_oov_indices to 0.
-            index = StringLookup(
-                vocabulary=vocabulary, mask_token=None, num_oov_indices=0
+            lookup = StringLookup(
+                vocabulary=vocabulary,
+                mask_token=None,
+                num_oov_indices=0,
+                output_mode="int" if use_embedding else "binary",
```

Comment:
I assume we'll keep this at "binary" in the future (which is totally fine even after we rename it to multi_hot, we can keep binary as a first-class alias)